tls/s2n_config.c

```diff
@@ -404,3 +405,16 @@ int s2n_config_set_client_hello_cb(struct s2n_config *config, s2n_client_hello_f
 
     return 0;
 }
+
+int s2n_config_set_max_fragment_length(struct s2n_config *config, uint8_t mfl_code)
+{
+    if (mfl_code >= S2N_TLS_MAX_FRAG_LEN_INVALID) {
+        fprintf(stderr, "invalid Max Fragmentation Length encountered\n");
```

Comment:
Let's use `S2N_ERROR(S2N_ERR_*)` instead of fprintf/return 0 to match our error return convention.